ui/app/pages/create-account/connect-hardware/select-hardware.js

```diff
@@ -117,74 +123,95 @@ export default class SelectHardware extends Component {
     );
   }
 
-  getAffiliateLinks() {
-    const links = {
-      trezor: `<a class='hw-connect__get-hw__link' href='https://shop.trezor.io/?a=metamask' target='_blank'>Trezor</a>`,
-      ledger: `<a class='hw-connect__get-hw__link' href='https://www.ledger.com/products/ledger-nano-s?r=17c4991a03fa&tracker=MY_TRACKER' target='_blank'>Ledger</a>`,
-    };
-
-    const text = this.context.t('orderOneHere');
-    const response = text
-      .replace('Trezor', links.trezor)
-      .replace('Ledger', links.ledger);
-
-    return (
-      <div
-        className="hw-connect__get-hw__msg"
-        dangerouslySetInnerHTML={{ __html: response }}
-      />
-    );
-  }
-
-  renderTrezorAffiliateLink() {
-    return (
-      <div className="hw-connect__get-hw">
-        <p className="hw-connect__get-hw__msg">
-          {this.context.t('dontHaveAHardwareWallet')}
-        </p>
-        {this.getAffiliateLinks()}
-      </div>
-    );
+  renderTutorialsteps() {
+    switch (this.state.selectedDevice) {
+      case 'ledger':
+        return this.renderLedgerTutorialSteps();
+      case 'trezor':
+        return this.renderTrezorTutorialSteps();
+      default:
+        return '';
+    }
   }
 
-  scrollToTutorial = () => {
-    if (this.referenceNode) {
-      this.referenceNode.scrollIntoView({ behavior: 'smooth' });
+  renderLedgerTutorialSteps() {
+    const steps = [];
+    if (this.props.useLedgerLive) {
+      steps.push({
+        title: this.context.t('step1LedgerWallet'),
+        message: this.context.t('step1LedgerWalletMsg', [
+          // eslint-disable-next-line react/jsx-key
+          <a
+            className="hw-connect__msg__link"
+            href="https://www.ledger.com/ledger-live"
+            rel="noopener noreferrer"
+            target="_blank"
+          >
+            Ledger Live App
```

Comment:
This requires localization

Reply:
You're probably correct that "App" isn't universal, updated.